README.md

```diff
@@ -90,6 +90,8 @@
 42. [**Youtube Multi Downloader**](https://youtubemultidownloader.com) - Download a complete playlist on youtube.
 43. [**Hotstar for Free**](http://kattappa.me/newlive?url=LINK_OF_HOTSTAR_VIDEO)- You can watch live match or anything else without having premium account on HotStar
 44. [**draw.io**](https://www.draw.io/) - Draw flowcharts, ER Diagrams easily.
+45. [**Hackerrank**](https://www.hackerrank.com/), [**Codechef**](https://www.codechef.com/) - Competitive Coding
```

Comment:
These are already very well know, so I don't think there is benefit in adding these.